packages/block-editor/CHANGELOG.md

```diff
@@ -2,63 +2,67 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+-   The block controll value for `InnerBlocks` has been changed from `__experimentalBlocks` to `value` and is now considered a stable API.
```

Comment:
```suggestion
-   The block control value for `InnerBlocks` has been changed from `__experimentalBlocks` to `value` and is now considered a stable API.
```

Not 100% sure if you meant `control` or `controlled` but a typo either way 😅

Reply:
Nice catch! i think `control` works.